pkgs/development/python-modules/flatbuffers/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv
+, buildPythonPackage
+, pkgs
+}:
+
+let
+  flatbuf = pkgs.flatbuffers;
+in buildPythonPackage rec {
+  pname = "flatbuffers";
+  version = flatbuf.version;
+  src = flatbuf.src;
+  sourceRoot = "source/python";
```

Some spacing would be nice.
```suggestion
  pname = "flatbuffers";
  version = flatbuf.version;
  
  src = flatbuf.src;
  sourceRoot = "source/python";
```

---

Will fix once the top issue is investigated.